content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,214 @@
+reveals the :guilabel:`Event Templates` page. Here, users can find all the existing event templates
+in the database.
+
+.. image:: event_templates/event-templates-page.png
+   :align: center
+   :alt: The Event Templates page in the Odoo Events application.
+
+By default, Odoo provides users with three pre-configured event templates: :guilabel:`Exhibition`,
+:guilabel:`Training`, and :guilabel:`Sport`. All of these pre-configured event templates that Odoo
+provides by default have their own unique customizations applied to them.
+
+To change how these event templates appear on the *Template* drop-down field on an event form, users
+can drag-and-drop them into any desired order, using the :guilabel:`six block` icons, located to the
+left of each event template line on the :guilabel:`Event Templates` page.
+
+Create event template
+=====================
+
+There are two ways users can create and configure an event template in Odoo *Events*.
+
+Either navigate to :menuselection:`Events app --> Configuration --> Event Templates`, and click the
+:guilabel:`New` button in the upper-left corner. Doing so reveals a blank event template form that
+can be customized in a number of different ways.
+
+Or, on an event form, start typing the name of a new event template in the *Template* field, and
+click :guilabel:`Create and edit...` from the resulting drop-down menu. Doing so reveals a *Create
+Template* pop-up window, featuring all the same configurable fields and elements found on a standard
+event template form.
+
+.. note::
+   Clicking :guilabel:`Create "[template name]"` from the resulting drop-down menu, via the
+   *Template* field on an event form creates the event template in the database, but does **not**
+   present the user with the *Create Template* pop-up window.
+
+   The event template would have to be modified, by selecting it on the *Event Templates* page
+   (:menuselection:`Events app --> Configuration --> Event Templates`).
+
+Event template form
+-------------------
+
+All the fields on a standard :guilabel:`Event Template` form are *also* on the *Create Template*
+pop-up window, accessible via the *Template* field on an event form.
+
+.. image:: event_templates/event-template-form.png
+   :align: center
+   :alt: A standard event template form in the Odoo Events application.
+
+Start by providing the event template with a name in the :guilabel:`Event Template` field, located
+at the top of the form.
+
+Beneath that field, there is a series of selectable checkboxes, all of which are related to how the
+event menu will be displayed on the event web page.
+
+- :guilabel:`Website Submenu`: Odoo presents an event-specific submenu of selectable options that
+  are related to that specific event. When this checkbox is ticked, every other checkbox in this
+  series is automatically ticked, as well. Users can then choose to untick (deselect) any of the
+  checkbox options, as desired.
```

Comment:
I completely understand the suggestion, however, since some readers may not be aware of the concept of a 'submenu' I am going to opt for the slightly more "elaborate" explanation here, just to ensure there is no confusion. That said, if I receive a similar comment about this, I'll be sure to implement your suggestion (and give you full credit 😄 )